src/EventStore.Core/Bus/InMemoryBus.cs

```diff
@@ -80,11 +80,11 @@ private async ValueTask DispatchAndWatchSlowMsg(MessageTypeHandler handlers, Mes
 		var elapsedMs = ts.ElapsedMilliseconds;
 		if (elapsedMs > _slowMsgThresholdMs) {
 			Log.Debug("SLOW BUS MSG [{bus}]: {message} - {elapsed}ms.",
-				Name, message.GetType().Name, elapsedMs);
+				Name, message.GetType().Name, (int)elapsedMs);
 			if (elapsedMs > QueuedHandlerThreadPool.VerySlowMsgThreshold.TotalMilliseconds &&
 			    message is not SystemMessage.SystemInit)
 				Log.Error("---!!! VERY SLOW BUS MSG [{bus}]: {message} - {elapsed}ms.",
-					Name, message.GetType().Name, elapsedMs);
+					Name, message.GetType().Name, (int)elapsedMs);
```

Comment:
were these changes meant to be in this PR?

Reply:
they aren't strictly related, but it was a recent small regression that this fixes